documentation/add/09_test_data_storage.md

```diff
@@ -0,0 +1,94 @@
+# Test data storage, distribution and requirements for PACE project
+
+Date: 3-11-2021
+
+## Current State
+Currently, the test data are stored on the GitHub Repo as part of the Herbert/Horace project. 
+Anyone who clones the GitHub repo also clones all test data and the history of binary mat files.
+
+All users of the GitHub version have direct access to the test data, stored in the correct location.
+
+### Issues
+* Cloning the repo can take excessive time and Repo is approaching 1GB in size. 
+  Because of their binary format, git cannot easily calculate the diffs and as such will store multiple full copies in the history.
+* In order to remedy this, a workaround has been performed 
+  (removing copies from the history) on a semi-regular basis which requires all branches to be saved, pulled and merged; 
+  this can result in corruption if care is not taken.
+* Future requirements of the project include benchmarking. 
+  Should benchmarking tests require large data (likely), these should not be stored in the repo for above reasons.
+
+## Requirements
+* It is not necessary that the data should be secure/private and does not require special considerations, however, write-access should be protected.
+* The data must be accessible to any Jenkins CI machines.
+* The data should be accessible to developer machines or at least IDaaS.
+* The data might want to be accessible to standard users to verify their PACE install.
+* Data should be programatically retrievable (particularly important for CI testing)
+  either through shell or through mounting virtual drive.
+
+
+# Appendix - Potential solutions to issues
+## FTP server
+Externally accessible FTP server to store and recover files. 
+
+### Pros
+* This can be made globally accessible to anyone who needs data.
+* Can be linked into CMake with possibility of only pulling test data if deemed necessary.
+
+### Cons
+* Requires management/support of external server.
+* Trickier to manage version control which may result in inconsistencies between current code state and test data.
+
+## SAN Storage
+Discussions have been held regarding migrating test data to the Storage Area Network (SAN). 
+Data can be moved across to the SAN and treated like local drive. 
+Instructions on how to gain access to the SAN can be found [here](https://github.com/pace-neutrons/Horace/blob/master/documentation/smg/10_how_to_use_the_SAN_area.md).
+
+\<Unsure what discussions have resulted in\>
+
+### Pros
+* SAN exists and managed.
+* Data appears as though local drive.
+* Don't necessarily have to store data copies on local machines.
+* Can be selective about data copied to local machine.
+
+### Cons
+* Inaccessible to external users to test install.
+* Can be complex to connect to SAN drive(?)
+* Issues with CI system connecting to remote drive(?)
```

Comment:
Should probably spell out what the question marks signify

Reply:
Sorry, question marks in brackets are my brief notation saying "I'm not sure if this is the case, can someone clarify". I realise that's not standard notation, but they're there to remind me to ask someone who knows.